src/main/webapp/app/overview/course-score-calculation.service.ts

```diff
@@ -36,8 +39,15 @@ export class CourseScoreCalculationService {
                             score = 0;
                         }
                         absoluteScore = absoluteScore + score * this.SCORE_NORMALIZATION_VALUE * exercise.maxScore;
+                        reachableScore += exercise.maxScore;
                     }
                     presentationScore += participation.presentationScore !== undefined ? participation.presentationScore : 0;
+
+                    if (participation.initializationState === InitializationState.INITIALIZED && exercise.type !== ExerciseType.PROGRAMMING) {
+                        reachableScore += exercise.maxScore;
```

Comment:
How do you handle programming exercises? I cant find a specific condition that match programming exercises?

It seems to work, I just could not figure out why, maybe you could add a comment for that?

Reply:
I believe programming exercises keep their initialized state even after they are done. Therefore the reachable score was added twice.